meters/rs485/sdm72.go

```diff
@@ -0,0 +1,62 @@
+package rs485
+
+import . "github.com/volkszaehler/mbmd/meters"
+
+func init() {
+	Register(NewSDM72Producer)
+}
+
+const (
+	METERTYPE_SDM72 = "SDM72"
+)
+
+type SDM72Producer struct {
+	Opcodes
+}
+
+func NewSDM72Producer() Producer {
+	/**
+	 * Opcodes as defined by Eastron SDM72.
+	 * See https://data.stromzähler.eu/eastron/SDM72DM-manual.pdf
+	 */
+	ops := Opcodes{
+		Power:  0x0034,
+		Import: 0x0048,
+		Export: 0x004a,
+		Sum:    0x0156,
+	}
+	return &SDM72Producer{Opcodes: ops}
+}
+
+func (p *SDM72Producer) Type() string {
+	return METERTYPE_SDM72
+}
+
+func (p *SDM72Producer) Description() string {
+	return "Eastron SDM72"
+}
+
+func (p *SDM72Producer) snip(iec Measurement) Operation {
+	operation := Operation{
+		FuncCode:  ReadInputReg,
+		OpCode:    p.Opcode(iec),
+		ReadLen:   2,
+		IEC61850:  iec,
+		Transform: RTUIeee754ToFloat64,
+	}
+	return operation
+}
+
+// This device does not provide voltage data
+// so it is not possible to automatically detect the device
+func (p *SDM72Producer) Probe() Operation {
```

Comment:
This will cause an NPE if register 0 can be read. Either extend signature to include an error if probing is not supported, make the `Probe` interface optional (more complicated) or check for the zero measurement in https://github.com/volkszaehler/mbmd/blob/e696b1dacca269b6b6cd145d27256423cd853105/meters/rs485/rs485.go#L95